aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/async-and-stored-procedures-with-the-entity-framework-in-an-asp-net-mvc-application.md

```diff
@@ -139,6 +139,10 @@ This section requires you to have completed the optional **Deploying the app to
 
     The first time you run a page that accesses the database, the Entity Framework runs all of the migrations `Up` methods required to bring the database up to date with the current data model. You can now use all of the web pages that you added since the last time you deployed, including the Department pages that you added in this tutorial.
 
+## Get the code
+
+You can go to the MSDN code gallery and [download the completed project](http://code.msdn.microsoft.com/ASPNET-MVC-Application-b01a9fe8).
```

Comment:
I'd rather not mention code gallery -- when I find a new home for the code I'd like to be able to do a global search and replace on the URL.

Reply:
Okay. I switched it back to: 

## Get the code

[Download the Completed Project](http://code.msdn.microsoft.com/ASPNET-MVC-Application-b01a9fe8)


If you'd like any other updates, I'm available.
Otherwise, this PR is ready to be merged.